ibm/service/scc/resource_ibm_scc_scope.go

```diff
@@ -0,0 +1,735 @@
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_scc_scope", "environment"),
+				Description:  "The scope environment. This value details what cloud provider the scope targets.",
+			},
+			// Manual Change: change name and value for scope_type and scope_id
+			"properties": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "A list of scopes/targets to exclude from a scope.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"account_id": &schema.Schema{
+							Type:        schema.TypeString,
+							ForceNew:    true,
+							Optional:    true,
+							Description: "The ID of the account to target",
+							ConflictsWith: []string{
+								"properties.0.enterprise_id",
+								"properties.0.resource_group_id",
+								"properties.0.ibm_facts_api_instance_id",
+								"properties.0.account_group_id"},
+						},
+						"enterprise_id": &schema.Schema{
+							Type:        schema.TypeString,
+							ForceNew:    true,
+							Optional:    true,
+							Description: "The ID of the enterprise to target",
+							ConflictsWith: []string{
+								"properties.0.account_id",
+								"properties.0.resource_group_id",
+								"properties.0.ibm_facts_api_instance_id",
+								"properties.0.account_group_id"},
+						},
+						"resource_group_id": &schema.Schema{
+							Type:        schema.TypeString,
+							ForceNew:    true,
+							Optional:    true,
+							Description: "The ID of the resource group to target",
+							ConflictsWith: []string{
+								"properties.0.account_id",
+								"properties.0.enterprise_id",
+								"properties.0.ibm_facts_api_instance_id",
+								"properties.0.account_group_id"},
+						},
+						"account_group_id": &schema.Schema{
+							Type:        schema.TypeString,
+							ForceNew:    true,
+							Optional:    true,
+							Description: "The ID of the account group to target",
+							ConflictsWith: []string{
+								"properties.0.account_id",
+								"properties.0.enterprise_id",
+								"properties.0.ibm_facts_api_instance_id",
+								"properties.0.resource_group_id"},
+						},
+						"ibm_facts_api_instance_id": &schema.Schema{
+							Type:        schema.TypeString,
+							ForceNew:    true,
+							Optional:    true,
+							Description: "The Security and Compliance Center provider type instance ID to target",
+							ConflictsWith: []string{
+								"properties.0.account_id",
+								"properties.0.enterprise_id",
+								"properties.0.account_group_id",
+								"properties.0.resource_group_id"},
+						},
+					},
+				},
+			},
+			"exclusions": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				Description: "A list of scopes/targets to exclude from a scope.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"account_id": &schema.Schema{
```

Comment:
This is again key value pair

{
					"name": "exclusions",
					"value": [
						{
							"scope_id": "1e351d2203ba44cb8a8bbe971fff7771",
							"scope_type": "enterprise.account_group"
						}
					]
				}